front_end/panels/ai_chat/core/EnvironmentConfig.ts

```
@@ -0,0 +1,336 @@
+  getStorageKey(provider: APIKeyProvider): string {
+    return STORAGE_KEYS[provider];
+  }
+
+  /**
+   * Validate credentials for a provider
+   * 
+   * @param provider The API key provider
+   * @returns Validation result with details
+   */
+  validateCredentials(provider: APIKeyProvider): {
+    isValid: boolean;
```

ESLint

[error] Expected a comma.

(@stylistic/member-delimiter-style)